smoke-tests/src/test/groovy/io/opentelemetry/smoketest/JettyJpmsSmokeTest.groovy

```diff
@@ -16,7 +16,7 @@ abstract class JettyJpmsSmokeTest extends AppServerTest {
   @Override
   protected String[] getCommand() {
     // --jpms flags enables using java module system
-    return ["java", "-jar", "/server/start.jar", "--jpms"]
+    return ["java", "-jar", "/server/start.jar", "--jpms", "-Dcom.sun.management.jmxremote"]
```

Comment:
just checking that this is an intentional change (no objection if it is)

Reply:
Thanks, you are absolutely right, this wasn't intentional.